messages_en-2.txt

```diff
@@ -572,15 +574,19 @@ sync.fail.bad.network=Couldn't contact server. Please make sure an internet conn
 sync.fail.server.error=The server had an error processing your data. Please try again later. If the problem persists, contact technical support.
 sync.fail.unknown=An unexpected issue occurred during sync. Please try to sync again.
 
-sync.progress.title=Communicating with Server
```

Comment:
I think these strings are used in 1 or 2 places: showing keys and default values in the excel translation file and populating `default/app_strings.txt` in cczs (I think..).
It makes a lot of sense for this file to be versioned and have HQ provide the correct version in those places, but I don't think that is something we have setup, so we always just serve the latest version. 

Could be wrong on all of this, it is one of those parts of the product that I'm not sure anyone fully understands...